Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -23,10 +23,117 @@
 from abc import abstractmethod
 from distutils.version import LooseVersion
 from threading import Lock
+from inspect import currentframe
 
 import demistomock as demisto
 import warnings
 
+
+def __line__():
+    cf = currentframe()
+    return cf.f_back.f_lineno
+
+
+# 39 - the line offset from the beggining of the file.
+_MODULES_LINE_MAPPING = {
+    'CommonServerPython': {'start': __line__() - 39, 'end': float('inf')},
```

Comment:
__line__() here is not 39?!!

Reply:
it is, but we want to set it as the first line of the module which is 39 lines above